plugins/module_utils/network/iosxr/config/l2_interfaces/l2_interfaces.py

```diff
@@ -192,6 +218,20 @@ def _state_overridden(self, want, have, module):
 
         return commands
 
+    '''
+    def _state_rendered(self, want):
```

Comment:
Why do we need a separate call for rendered? Also, this seems to be commented out?

Reply:
@NilashishC yeah seems this commented code needs to be removed actually, thanks